features/events/traps/blueprint-trapd-listener.xml

```diff
@@ -11,16 +11,32 @@
         http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
 ">
 
-    <cm:property-placeholder id="trapdProperties" persistent-id="org.opennms.netmgt.trapd" update-strategy="reload">
+    <cm:property-placeholder id="trapHandlerDefaultProperties" persistent-id="org.opennms.netmgt.trapd" update-strategy="none">
         <cm:default-properties>
             <cm:property name="trapd.listen.interface" value="127.0.0.1" />
-            <cm:property name="trapd.listen.port" value="162" />
+            <cm:property name="trapd.listen.port" value="5150" />
         </cm:default-properties>
     </cm:property-placeholder>
 
+	<!-- Camel Event for unmarshalling processor -->
+	<bean id="unmarshaller" class="org.opennms.core.camel.JaxbUtilsUnmarshalProcessor">
+		<argument value="org.opennms.netmgt.config.trapd.TrapdConfiguration" />
+	</bean>
+
+	<camelContext id="trapConnectionHandlerDefaultContext"
+		xmlns="http://camel.apache.org/schema/blueprint">
+		<propertyPlaceholder id="properties"
+			location="blueprint:trapHandlerDefaultProperties" />
+		<route id="unmarshal">
+			<from uri="http://localhost:8980/opennms/rest/config/trapd" />
+			<bean ref="unmarshaller" />
+			<to uri="bean:trapdConfig?method=onSnmpV3UsersUpdate" />
```

Comment:
In addition to reloading the config, I think that the route also needs to restart the listener with the new list of SNMPv3 users but only when the configuration has changed. Please investigate this and confirm whether this is necessary.